setup.py

```diff
@@ -0,0 +1,97 @@
+"""
+This module defines the attributes of the
+PyPI package for the mbed SDK test suite
+"""
+
+"""
+mbed SDK
+Copyright (c) 2011-2021 ARM Limited
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+Author: Przemyslaw Wirkus <Przemyslaw.Wirkus@arm.com>
```

Comment:
Do we need to call out the author's name here?

Reply:
Removed